packages/amplify_api_plugin_interface/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## [0.0.1] - TODO: Add release date.
+
+* TODO: Describe initial release.
```

Comment:
Must remember to update upon release.

Reply:
Updated. Will add the date at the time of the release